docs/ops/movement/Unique_10.md

```diff
@@ -0,0 +1,119 @@
+# Unique  {#openvino_docs_ops_comparison_Unique_10}
```

Comment:
I think you also need to create opset10.md file and add reference to opset10 from the content.